### ISSUE TITLE
Can't drag to select in TextField on iPad

### ISSUE THREAD

**Issue by Renzo-Olivares** [flutter/flutter#105639]
Dragging to select on a TextField should select the range of text from the position where the drag started to the position where the drag moves to. On iPad it selects 1-2 characters and stops.

https://user-images.githubusercontent.com/948037/172697035-39036d12-a1e1-4d1d-8295-02010af12cf5.mov

### To reproduce

  1. Run any app with text field, such as the one below.
  2. Focus the field and enter a couple of lines of text.
  3. Drag from one position in the text to another position.

Expected: The text between the start position of the drag and moved position of the drag is selected.
Actual: 1-2 characters are selected.

<details>

<summary>Code</summary>

```dart
import 'package:flutter/material.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
        useMaterial3: true,
      ),
      home: const MyHomePage(title: 'Flutter Demo Home Page'),
    );
  }
}

class MyHomePage extends StatefulWidget {
  const MyHomePage({Key? key, required this.title}) : super(key: key);

  final String title;

  @override
  State<MyHomePage> createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  final TextEditingController _controller = TextEditingController(text: 'Atwater Peel Sherbrooke Bonaventure');

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text(widget.title),
      ),
      body: Center(
        child: TextField(
          controller: _controller,
        ),
      ),
    );
  }
}
```

</details>

**Comment by Renzo-Olivares** [comment]
Tried to bisect this, and it seems its been broken for a while. Went back to #e40d7e6ef40e2466d5be57c9633fd50376140b65 but it was still not working.

**Comment by Renzo-Olivares** [comment]
Tried with an iPhone 13 Pro Max with a mouse connected via bluetooth, with "perform touch gestures" disabled and am also seeing drag to select broken.

**Comment by Renzo-Olivares** [comment]
Looks like this is working as expected now on `master` when using `capture pointer device` on iPad simulator.

https://github.com/user-attachments/assets/8a2748d3-28b0-47aa-963b-99b6ea2dd766

Also tested on a physical iPhone on iOS 26.0, and the flutter behavior matches the one on the system when using "perform touch gestures" under AssistiveTouch settings. When disabling that option iOS behaves similarly to iPad with a mouse on Flutter with regards to text selection (desktop level text selection gestures). The native iOS text selection behavior whether the touch gestures option is enabled/disabled stays the same. I think this might actually be a pro on Flutter's end since our text selection with mouse behaves as it would on a desktop, rather than just converting the mouse events to touch events, for example a click + drag actually selects text on Flutter where a click + drag on iOS does nothing. @loic-sharma @justinmc do you guys have any thoughts on this behavior discrepancy.

**Comment by loic-sharma** [comment]
Yup that discrepancy sounds good to me! Thanks for testing this :)
